### ISSUE TITLE
Use Better Ricin Icon

### ISSUE THREAD

**Issue by gitgud-software** [RicinApp/Ricin#160]
If this exists:

![image](https://cloud.githubusercontent.com/assets/10673048/20652701/12ef594a-b4cc-11e6-8537-2c3c7d79621e.png)

why are we still using this everywhere in the application still?

![image](https://raw.githubusercontent.com/RicinApp/Ricin/master/res/images/icons/ricin_logo.png)

The former is much better, provided there is a scalable version of it.

**Comment by SkyzohKey** [comment]
Yeah I know that the first logo was probably better but since the original graphist didn't make any sign of life since he made it, and no scalable version of it is available, I'll keep using the second one.

Anyway, If you are interested in making the same as first one but in SVG, feel free to open a pull-request ;)
